fairmq/devices/FairMQBenchmarkSampler.cxx

```diff
@@ -1,8 +1,8 @@
 /********************************************************************************
  *    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
```

Comment:
Let's also update the year, e.g.

` * Copyright (C) 2014-2018 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH *`

Reply:
ok, will do.